tests/brevitas/core/bit_width_fixture.py

```diff
@@ -81,4 +92,4 @@ def bit_width_parameter(bit_width_init, min_bit_width_init, override_pretrained)
 
 
 # Union of all variants of bit-width
-fixture_union('bit_width_all', ['bit_width_const', 'bit_width_parameter'])
+fixture_union('bit_width_all', ['bit_width_const', 'bit_width_parameter', 'bit_width_stateful_const'])
```

Comment:
I would also add a separate `bit_width_stateful` union with stateful_const and parameter, and then update the test `test_ignore_missing_keys` to take bit_width_stateful.